app/views/posts/index.html.erb

```diff
@@ -28,5 +31,7 @@
    </div>
  <% end %>
 
- <button class="pagination">Pagination</button>
+ <button class="pagination">
+ <%= link_to 'Pagination', user_post_path(@user, @posts[0]) %>
+ </button>
```

Comment:
- [ ] At the moment, when I click on a user to see their posts, I get this error page. Please kindly help fix it.

- [ ] Also please kindly ensure that the like action and creating comments work as expected.

![image](https://user-images.githubusercontent.com/43995830/219688370-ca07de53-5592-4bb2-84f6-938bd78793cf.png)